app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -62,9 +62,10 @@ func newInstallControlPlaneCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 		SdsTlsKey:               "",
 	}
 	cmd := &cobra.Command{
-		Use:   "control-plane",
-		Short: "Install Kuma Control Plane on Kubernetes",
-		Long:  `Install Kuma Control Plane on Kubernetes.`,
+		Use:     "control-plane",
+		Short:   "Install Kuma Control Plane on Kubernetes",
+		Long:    `Install Kuma Control Plane on Kubernetes.`,
+		Example: `1. kumactl install control-plane | kubectl apply -f -`,
```

Comment:
Sorry, my example with `kumactl install database-schema` was misleading.

Let's use a slightly different format for `Example` field:
* Let's add a line with a short description for each example command
* Let's provide 2-3 examples for each command, e.g.
  * 1st example is the shortest or the most practical option
  * 2nd example to demonstrate how to override default values

E.g.,

```
	cmd := &cobra.Command{
		Use:   "control-plane",
		Short: "Install Kuma Control Plane on Kubernetes",
		Long:  `Install Kuma Control Plane on Kubernetes.`,

		Example: `
  # Install Kuma on Kubernetes
  kumactl install control-plane | kubectl apply -f -

  # Generate installation script for a target platform. At the moment, always generate Kubernetes YAML
  kumactl install control-plane`,

  		RunE: func(cmd *cobra.Command, _ []string) error {
                     ...
                },
	}
```
which will produce the following output:

```
Install Kuma Control Plane on Kubernetes.

Usage:
  kumactl install control-plane [flags]

Examples:

  # Install Kuma on Kubernetes
  kumactl install control-plane | kubectl apply -f -

  # Generate installation script for a target platform. At the moment, always generate Kubernetes YAML
  kumactl install control-plane

Flags:

  ...
```